readme/category_resource_test.go

```diff
@@ -229,6 +231,63 @@ func TestCategoryResource_Validation_Error(t *testing.T) {
 	})
 }
 
+// TestCategoryResource_ReCreate tests that a category gets re-created if it is deleted outside of Terraform.
+func TestCategoryResource_ReCreate(t *testing.T) {
+	// Close all gocks after completion.
+	defer gock.OffAll()
```

Comment:
are there any hooks to "afterEach test" that could be useful to leverage for turning gock off after test completion?

i may be way off base, just wondering if this is something foundational to use in developing a good pattern